cfme/fixtures/utility_vm.py

```diff
@@ -32,29 +32,40 @@
 from cfme.utils.wait import TimedOutError
 
 
+# I know this isn't good for reuse, I am just trying to get something to work.
 def _trying_ips(vm, attempts=60, interval=10):
     for attempt in range(attempts):
         for ip in getattr(vm, 'all_ips', []):
-            yield ip
-            return
+            try:
+                if ip.split('.')[0] == '10':
+                    yield ip
+                    return
+            except AttributeError:
+                pass
         time.sleep(interval)
 
 
 @contextmanager
 def connect_ssh(vm, creds):
     for ip in _trying_ips(vm):
-        try:
-            with SSHClient(hostname=ip, username=creds.principal, password=creds.secret) as client:
-                logger.info("SSH connected to IP %s", ip)
-                result = client.run_command("true")
-                if not result.success:
-                    raise Exception(f"Command `true` failed on ip {ip}.")
-                yield client
-                return
-        except Exception as ex:
-            logger.info("Failed to connect with SSH to %s: %s", ip, ex)
+        for ssh_attempt in range(55):  # I know ... hardcoded. I'll fix
```

Comment:
Not good as the _trying_ips already does 60 attempts by default

Reply:
Doesn't the _trying_ips just do 60 attempts to get an IP? Once we get one, we need to try to connect via SSH until the SSH client is up (until we are successful).